specification/attributes/numeric_format.md

```diff
@@ -0,0 +1,72 @@
+* Numeric values MUST NOT include a character to represent a sign for a positive value. A negative sign (-) MUST indicate a negative value.
+* Numeric values MUST NOT be used to represent binary values (e.g., 0 or 1).
+* Columns with a Numeric value format MUST present one of the following values as the "Data type" in the column definition.
+  | Data Type | Type Description |
+  |:----------|:-----------------|
+  | Numeric   | Specifies any numeric value compliant with this attribute definition. This type is used when the column definition does not specify a more specific numeric type. |
+  | Integer   | Specifies a numeric value represented by a whole number or by zero. Integer number formats correspond to standard data types defined by ISO/IEC 9899:2018 |
+  | Decimal   | Specifies a numeric value represented by a decimal number. Decimal formats correspond to ISO/IEC/IEEE 60559:2011 and IEEE 754-2008 definitions. |
+* Providers SHOULD define one of the following precision values for every column with a numeric data type in a data definition document that providers publish.
+  | Data Type | Precision | Definition                                                                | Range / Significant Digits       |
+  |:----------|:----------|:--------------------------------------------------------------------------|:---------------------------------|
+  | Integer   | Short     | 16-bit signed short int ISO/IEC 9899:2018                                 | -32,767 to +32,767               |
+  | Integer   | Long      | 32-bit signed long int ISO/IEC 9899:2018                                  | -2,147,483,647 to +2,147,483,647 |
+  | Integer   | Extended  | 64-bit signed two's complement integer *or higher*                        | -(2^63 - 1) to (2^63 - 1)        |
+  | Decimal   | Single    | 32-bit binary format IEEE 754-2008 floating-point (decimal32)             | 9                                |
+  | Decimal   | Double    | 64-bit binary format IEEE 754-2008 floating-point (decimal64)             | 16                               |
+  | Decimal   | Extended  | 128-bit binary format IEEE 754-2008 floating-point (decimal128) or higher | 36+                              |
+
+### Examples
+
+This format requires that single numeric values be represented using an integer or decimal format without additional characters or qualifiers. The following lists provide examples of values that meet the requirements and those that do not.
+
+* Values Meeting Numeric Requirements:
+  * -100.2
+  * -3
+  * 4
+  * 1.234
+  
+* Values NOT Meeting Numeric Requirements
+  * 1 1/2 - contains fractional notation
+  * [3,5,8] - contains an array
+  * [4:5] - contains a range
+  * 5i + 4 - contains a complex number
+  * sqrt(2) - contains a mathematical symbol or operation
+  * 2.3^3 - contains an exponent
+  * 32 GiB - contains a unit of measure
+  * $32 - contains a currency symbol
+  * 3,432,342 - contains a comma
+  * +333 - contains a positive sign
+
```

Comment:
E notation is prohibited or? I suggest including it as one of the examples.

Reply:
I understand from the requirements that the E/e exponential notation is disallowed.
This notation is supported by JSON, Python and many languages.
This notation is used in Azure billing data.
What is the rationale for disallowing it?

Reply:
I wasn't aware that Azure allowed this.  @marc-perreaut - I just committed language to allow it, and to specify that values must adhere to the E notation format.  Thanks for catching this.